### ISSUE TITLE
The ability to transfer event to different chapter 

### ISSUE THREAD

**Issue by Sboonny** [freeCodeCamp/chapter#1927]
### Discuss your ideas or share your views: 
Instance_owner can open event in any chapter, which may cause mistake (opening one in different chapter). 

They need to input the Event data again in different chapter, to create the same Event, and the chapter will send email about it cancelation, while it still available but in different chapter.

We can create a transfer function to take care of these issue automatically.

Kudos to Oliver for the awesome idea 👍

**Comment by gikf** [comment]
While thinking on this, I thought it might be useful to give user ability to change some event details, before creating _transferred_ event. That might be some intermediate feature, to use existing event to pre-fill form.

**Comment by Sboonny** [comment]
Good point, we may want to set the venue to TBD as default as well 🤔, they are part of chapter as far as I remember.

**Comment by allella** [comment]
I wouldn't hold up the MVP for this feature and it looks like it's already on the Roadmap, so perhaps close it for now and revisit post-MVP?

I guess an alternative to a transfer would be to show the organizer a confirmation warning, or page, showing critical values that can't be changed after creation, like the _chapter_.

Eventually, we may also allow for different _organizations_ on the same _instance_, so a warning or transfer could eventually cover the _organization_ value too.
